docs/docsite/rst/scenario_guides/vmware_rest_scenarios/task_outputs/Add_a_floppy_disk_drive.task.yaml

```diff
@@ -0,0 +1,5 @@
+- name: Add a floppy disk drive
+  vcenter_vm_hardware_floppy:
+    vm: '{{ test_vm1_info.id }}'
```

Comment:
please use FQCN here and throughout all the examples.

Reply:
I plan to do it, but in another iteration.

c.f. https://github.com/ansible-collections/vmware.vmware_rest/issues/120